app/[locale]/layout.jsx

```diff
@@ -13,7 +15,9 @@ const RootLayout = ({ children }) => {
   return (
     <html lang={locale}>
       <body>
-        <I18nProvider locale={locale}>{children}</I18nProvider>
+        <Theme>
```

Comment:
I think we can create a `Providers` component to put all client side provider in the same place

Reply:
Like the Theme, right?